### ISSUE TITLE
Tree & TreeSelect: p-tree-node-toggle-button missing aria-label

### ISSUE THREAD

**Issue by TaneliTuomola** [primefaces/primeng#17463]
### Describe the bug

The toggleAriaLabel input [exists in the Tree component](https://github.com/primefaces/primeng/blob/master/packages/primeng/src/tree/tree.ts#L902) but has not been used. The same input should also be added to the TreeSelect component, which uses the Tree component to display the tree.

### Pull Request Link

#17464 

### Reason for not contributing a PR

- [ ] Lack of time
- [ ] Unsure how to implement the fix/feature
- [ ] Difficulty understanding the codebase
- [ ] Other

### Reproducer

https://primeng.org/treeselect

### Environment

https://primeng.org/treeselect

### Angular version

19

### PrimeNG version

v19

### Steps to reproduce the behavior

https://primeng.org/treeselect
Check toggleBtn html.

### Expected behavior

Aria-label is added to the toggle button.

**Comment by TaneliTuomola** [comment]
> This issue has been automatically marked as stale due to a lack of activity. **If this issue is still relevant to you, please leave a comment** so we can keep it open. We apologize for not being able to prioritize it sooner. If you have any new information or questions, please share them in your comment!


The problem still exists (v21).
